examples/Generic_DataCapture.m

```diff
@@ -0,0 +1,35 @@
+%% Script for capturing buffers of data from a connected IIO interface
+
+% Instantiate the Generic Rx system object, and specify the device name and uri 
+rx = adi.Generic.Rx('ad4630-24', 'ip:analog.local');
+
+rx.SamplesPerFrame = 4096;  % Using values less than 3660 can yield poor 
+% performance, generally
+
+% Enable channels for data capture
+rx.EnabledChannels = [1];
+
+% Display device and channel attribute names
```

Comment:
I thought to leave these here so that when running the example script out of box for a specific part, the attributes are also printed out for the user to take note of and use subsequently. Anyone using the generic example would definitely also be able to modify the flow as per their needs.